src/main/java/core/model/ShoppingCart.java

```diff
@@ -0,0 +1,37 @@
+package core.model;
+
+import java.util.List;
+
+public class ShoppingCart {
+    private Long id;
+    private Long userId;
+    private List<Product> products;
+
+    public ShoppingCart(List<Product> products) {
+        this.products = products;
+    }
```

Comment:
same: here you will receive only `userId` in the constructor
initialize `products` as a new ArrayList